### DISCUSSION TITLE
Kibana reports

### DISCUSSION THREAD

**Discussion by domlaz11** [Security-Onion-Solutions/securityonion#6951] [General] [answered]
Hi,

I'm trying to make a report in  Kibana but i am getting this error : The "open URL" phase took longer than 60 seconds. You may need to increase "xpack.reporting.capture.timeouts.openUrl"

Where would I change this setting?

Do I need to add the config in securityonion_standalone.sls ?

Thank you

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/kibana.html#configuration
